pr_agent/algo/ai_handler.py

```diff
@@ -99,28 +109,39 @@
                 )
             if self.azure:
                 model = 'azure/' + model
+            system = get_settings().get("CONFIG.MODEL_SYSTEM_PREFIX", "") + system + \
```

Comment:
**Suggestion:** Consider using f-strings for string concatenation as it is more readable and efficient. [best practice]
```suggestion
            system = f"{get_settings().get('CONFIG.MODEL_SYSTEM_PREFIX', '')}{system}{get_settings().get('CONFIG.MODEL_SYSTEM_SUFFIX', '')}"
            user = f"{get_settings().get('CONFIG.MODEL_USER_PREFIX', '')}{user}{get_settings().get('CONFIG.MODEL_USER_SUFFIX', '')}"
```